benchmarks/test_correlation.py

```diff
@@ -17,10 +17,9 @@ def set_up(request):
     dirdat = os.path.dirname(__file__) + '/data/'
     cosmo = ccl.Cosmology(Omega_c=0.30, Omega_b=0.00, Omega_g=0, Omega_k=0,
                           h=0.7, sigma8=0.8, n_s=0.96, Neff=0, m_nu=0.0,
-                          w0=-1, wa=0, transfer_function='bbks',
+                          w0=-1, wa=0, T_CMB=2.7, transfer_function='bbks',
```

Comment:
Was T_CMB added in this PR?

Reply:
Yes. We were not updating in properly when mucking with the value for the parameters.